epa-under-siege-peer-reviews.html

```diff
@@ -9,6 +9,17 @@
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
         <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
         <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
+        <meta property="og:title" content="Part 1 - Peer Reviews"/>
+        <meta property="og:type" content="article"/>
+        <meta property="og:url" content="https://100days.envirodatagov.org/epa-under-siege-peer-reviews.html"/>
+        <meta property="og:image" content="assets/ch-1cover.jpg"/>
+        <meta property="og:description" content="The EPA Under Siege: Trump’s Assault in History and Testimony"/>
+        <meta name="twitter:site" value="@EnviroDGI">
+        <meta property="twitter:title" content="Part 1 - Peer Reviews"/>
```

Comment:
I believe Twitter cards use "name" not "property":
https://dev.twitter.com/cards/getting-started

> Open Graph protocol also specifies the use of `property` and `content` attributes for markup (`<meta property="og:image" content="http://example.com/ogp.jpg"/>`) while Twitter cards use `name` and `content`. 

This is mentioned next:

> Twitter’s parser will fall back to using property and content, so there is no need to modify existing Open Graph protocol markup if it already exists.

But I believe, as demonstrated in the example below it on the source page, that's just in reference to that fact that `og:*` properties can use used instead, and twitter will parse those for certain properties, as indicated on the markup tag reference page (the "OpenGraph" column): https://dev.twitter.com/cards/markup